mailbox/client_conn.go

```diff
@@ -86,33 +86,33 @@ const (
 	sendSocketTimeout = 1000 * time.Millisecond
 )
 
-// ConnStatus is a description of the connection status of the client.
-type ConnStatus string
+// ClientConnStatus is a description of the connection status of the client.
+type ClientConnStatus string
 
 const (
-	// ConnStatusNotConnected means that the client is not connected at all.
-	// This is likely due to the mailbox server being down.
-	ConnStatusNotConnected ConnStatus = "Not Connected"
+	// ClientConnStatusNotConnected means that the client is not connected
+	// at all. This is likely due to the mailbox server being down.
+	ClientConnStatusNotConnected ClientConnStatus = "Not Connected"
```

Comment:
To make these constants a bit less clunky, should we just use `ClientNotConnected`, `ClientSessionNotFound`? Or `ClientStatusNotConnected`, `ClientStatusSessionNotFound`?